cmd/kube-proxy/app/server_others.go

```diff
@@ -578,15 +578,10 @@ func tryIPVSProxy(canUseIPVS bool, kcompat iptables.KernelCompatTester) string {
 
 func tryIPTablesProxy(kcompat iptables.KernelCompatTester) string {
 	// guaranteed false on error, error only necessary for debugging
-	useIPTablesProxy, err := iptables.CanUseIPTablesProxier(kcompat)
+	_, err := iptables.CanUseIPTablesProxier(kcompat)
 	if err != nil {
 		utilruntime.HandleError(fmt.Errorf("can't determine whether to use iptables proxy, using userspace proxier: %v", err))
```

Comment:
We should fix the return value of `iptables.CanUseIPTablesProxier` if we were going to change this (ie, remove the boolean and only have an error). But really, `iptables.CanUseIPTablesProxier` is pointless. All it checks is whether the kernel has the `route_localnet` sysctl, which was added in 3.6, 10 years ago. We can just remove the checks entirely and assume that all kernels that can run kubernetes can run the iptables proxier.